### PR TITLE
Add `startWebRequestSync` to App Lab (hidden from the toolbox)

### PR DESCRIPTION
**Pull request by joshlory**
Sample usage:

```js
button("btn", "load");
textLabel("txt", "");

onEvent("btn", "click", function(event) {
  setText("txt", "Loading...");
  
  var result = startWebRequestSync("https://api.openweathermap.org/data/2.5/weather?q=Chicago");
  setText("txt", result);
});
```

### CODE REVIEW

File: apps/src/applab/commands.js [hunk 1]
```diff
@@ -1362,6 +1362,14 @@ applabCommands.startWebRequest = function (opts) {
   req.send();
 };
 
+applabCommands.startWebRequestSync = function (opts) {
+  applabCommands.startWebRequest({...opts,
+    func: (status, contentType, responseText) => {
+      opts.func(responseText);
+    }
+  });
+};
```
Comment (islemaster) [inline apps/src/applab/commands.js:1371]: What happens if the web request fails?  It seems like it should throw - do we get that for free with `nativeIsAsync` or do we need to handle it here where we're wrapping the callback?

**Comment by joshlory** [comment]
Request from @gtw!

File: apps/src/applab/commands.js (hunk 1 above)
Comment (joshlory) [reply apps/src/applab/commands.js:1371]: The behavior of `startWebRequest` is that it calls the callback with the error.  I matched that behavior for the synchronous version (the error message is returned to the caller).

Comment (joshlory) [reply apps/src/applab/commands.js:1371]: image

**Pull request merged**